addons/Paragraph/src/presenter.js

```diff
@@ -328,6 +347,10 @@ function AddonParagraph_create() {
         return presenter.upgradeAttribute(model, "Editable placeholder", "");
     };
 
+    presenter.upgradeWeight = function (model) {
+        return presenter.upgradeAttribute(model, "Weight", "");

```

Comment:
Wszystko ok, jedynie chciałbym zwrócić uwagę na to, że jeśli w lekcji mamy stary addon, to w property weight jest wpisywane "" . Nie wiem czy nie trzeba będzie wpisać 0.